src/gui/screen_printing.cpp

```diff
@@ -404,6 +376,10 @@ int screen_printing_event(screen_t *screen, window_t *window, uint8_t event, voi
         close_popup_message(screen);
     }
 
+    /*    if (Lock::IsLocked())
```

Comment:
@RadekVana do you have prepared some cleaning-up of the commented code here in the GUI?

Reply:
similar to previous conversation - i thing it is better to keep dead/unused code for reference and remove later.

Reply:
Lock part should not be needed, since after Roberts change there are no more guiloops.
I do not care about that commented code.
To answer @DRracer, no I do not.